sql: add doc for admin show ddl jobs (#857)
pingcap/docs: sql/admin.md

    | 38     | test    | t1         | create table  | public               | 32        | 37       | 0         | 2019-01-10 12:32:41.956 +0800 CST | synced        |
    | 36     | test    |            | drop table    | none                 | 32        | 34       | 0         | 2019-01-10 11:29:59.982 +0800 CST | synced        |
    | 35     | test    |            | create table  | public               | 32        | 34       | 0         | 2019-01-10 11:29:40.741 +0800 CST | synced        |
    | 33     | test    |            | create schema | public               | 32        | 0        | 0         | 2019-01-10 11:29:22.813 +0800 CST | synced        |
    +--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
    ```

    * `JOB_ID`: each DDL operation corresponds to one DDL job. `JOB_ID` is globally unique.
    * `DB_NAME`: the name of the database on which the DDL operations are performed.
    * `TABLE_NAME`: the name of the table on which the DDL operations are performed.
    * `JOB_TYPE`: the type of the DDL operations.

>     * `SCHEMA_STATE`: the current state of the schema. If the `JOB_TYPE` is `add index`, it's the state of the index; if the `JOB_TYPE` is `add column`, it's the state of the column; if the `JOB_TYPE` is `create table`, it's the state of the table. The common states include:

Reviewer: ```suggestion
    * `SCHEMA_STATE`: the current state of the schema. If the `JOB_TYPE` is `add index`, it is the state of the index; if the `JOB_TYPE` is `add column`, it is the state of the column; if the `JOB_TYPE` is `create table`, it is the state of the table. The common states include:
```

>         * `none`: it indicates not existing. When the drop or create operation fails and rolls back, it usually becomes the `none` state.

Reviewer: ```suggestion
        * `none`: it indicates not existing. When the `drop` or `create` operation fails and rolls back, it usually becomes the `none` state.
```

>         * `delete only`, `write only`, `delete reorganization`, `write reorganization`: these four states are intermediate states. For details, see the paper [Online, Asynchronous Schema Change in F1](http://static.googleusercontent.com/media/research.google.com/zh-CN//pubs/archive/41376.pdf). These states cannot be seen in common operations, because the conversion from the intermediate states is so quick. Only in `add index` operations, you can see the `write reorganization` state, which means that the index data is being added. 

Reviewer: ```suggestion
        * `delete only`, `write only`, `delete reorganization`, `write reorganization`: these four states are intermediate states. For details, see the paper [Online, Asynchronous Schema Change in F1](http://static.googleusercontent.com/media/research.google.com/zh-CN//pubs/archive/41376.pdf). These states cannot be seen in common operations, because the conversion from the intermediate states is so quick. You can see the `write reorganization` state only in `add index` operations, which means that the index data is being added.
```

        * `public`: it indicates existing and usable. When operations like `create table` and `add index/column` are finished, it usually becomes the `public` state, which means that the created table/column/index can be normally read and written now.
    * `SCHEMA_ID`: the ID of the database on which the DDL operations are performed.
    * `TABLE_ID`: the ID of the table on which the DDL operations are performed.
    * `ROW_COUNT`: the number of the data rows that have been added when running the `add index` operation.
    * `START_TIME`: the start time of the DDL operations.
    * `STATE`: the state of the DDL operations. The common states include:

>         * `none`: it indicates that the operation task has been put in the DDL job queue but hasn't been performed yet, because it is waiting for the previous tasks to complete. Another reason might be that it becomes the `none` state after running the drop operation, but it will soon be updated to the `synced` state, which means that all TiDB instances have been synced to this state.

Reviewer: ```suggestion
        * `none`: it indicates that the operation task has been put in the DDL job queue but has not been performed yet, because it is waiting for the previous tasks to complete. Another reason might be that it becomes the `none` state after running the `drop` operation, but it will soon be updated to the `synced` state, which means that all TiDB instances have been synced to this state.
```

        * `running`: it indicates that the operation is being performed.
        * `synced`: it indicates that the operation has been performed successfully and all TiDB instances have been synced to this state.

>         * `rollback done`: it indicates that the operation has failed to perform and has rolled back.

Reviewer: ```suggestion
        * `rollback done`: it indicates that the operation has failed and has finished rolling back.
```

>         * `rollingback`: it indicates that the operation has failed to perform and is rolling back.

Reviewer: ```suggestion
        * `rollingback`: it indicates that the operation has failed and is rolling back.
```

>         * `cancelling`: it indicates that the operation is being cancelled. This state only occurs when cancelling DDL tasks using the `ADMIN CANCEL DDL JOBS` command.

Reviewer: ```suggestion
        * `cancelling`: it indicates that the operation is being cancelled. This state only occurs when you cancel DDL tasks using the `ADMIN CANCEL DDL JOBS` command.
```
